apps/frontend/src/components/global/upload_tabs/EditEvaluationModal.vue

```diff
@@ -0,0 +1,187 @@
+                <v-btn color="blue darken-1" text @click="commitTag()">
+                  Save
+                </v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+          <v-data-table
+            :headers="headers"
+            :items="activeEvaluation.evaluationTags"
+            disable-sort
+          >
+            <template #[`item.value`]="{item}">
+              <v-edit-dialog
+                :return-value.sync="item.value"
+                large
+                @save="updateTag(item)"
+              >
+                <div>{{ item.value }}</div>
+                <template #input>
+                  <div class="mt-4 title">Update {{ item.value }}</div>
+                  <v-text-field
+                    v-model="item.value"
+                    label="Edit"
+                    single-line
+                    autofocus
+                  />
+                </template>
+              </v-edit-dialog>
+            </template>
+            <template #[`item.actions`]="{item}">
+              <v-icon small @click="deleteTag(item)">mdi-delete</v-icon>
+            </template>
+          </v-data-table>
+        </v-container>
+      </v-card-text>
+
+      <v-card-actions>
+        <v-spacer />
+        <v-btn
+          color="blue darken-1"
+          text
+          blue
+          @click.native="$emit('closeEditModal')"
+        >
+          Cancel
+        </v-btn>
+        <v-btn color="darken-1" text @click="updateEvaluation()">Save</v-btn>
+      </v-card-actions>
+    </v-card>
+  </Modal>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import Modal from '@/components/global/Modal.vue'
+import {Prop} from 'vue-property-decorator';
+import {SnackbarModule} from '@/store/snackbar';
+import {EvaluationModule} from '@/store/evaluations';
+import {IEvaluation} from '@heimdall/interfaces';
+import axios from 'axios';
+
+@Component({
+  components: {
+    Modal
+  }
+})
+export default class EditEvaluationModal extends Vue {
+  @Prop({default: false}) visible!: boolean;
+  @Prop({type: Object, required: true}) readonly active!: IEvaluation;
+
+  newTagDialog: boolean = false;
+  deleteTagDialog: boolean = false;
+  awaitingFinishTyping: boolean = false;
+  activeTag: any = {
+    id: '-1',
+    value: ''
+  };
+  activeEvaluation: IEvaluation = {...this.active}
+
+  headers: Object[] = [
+    {
+      text: 'Tag',
+      value: 'value'
+    },
+    {
+      text: 'Actions',
+      value: 'actions'
+    }
+  ];
+
+  async deleteTag(tag: any) {
+    await axios.delete(`/evaluation-tags/${tag.id}`).then((response) => {
+      this.activeEvaluation.evaluationTags!.splice(
+        this.activeEvaluation.evaluationTags!.indexOf(tag),
+        1
+      )
+      SnackbarModule.notify("Deleted tag successfully.")
+    }).catch((error) => {
+      SnackbarModule.HTTPFailure(error)
+    });
+  }
+
+  async updateTag(tag: any) {
```

Comment:
For consistency here can the type of this be changed to `EvaluationTag` and all the places where it is used updated accordingly?